src/tasks/initialization.cr

```diff
@@ -32,7 +32,7 @@ module PlaceOS::Tasks::Initialization
     metrics_config = {"metrics" => JSON::Any.new(metrics_url)}
 
     start_lock.synchronize do
-      authority = Entities.create_authority(name: application_name, domain: application_base, config: metrics_config)
+      authority = Entities.create_authority(name: application_base, domain: application_base, config: metrics_config)
```

Comment:
I think this can remain as `application_name` with the fix in the entities initializer

```suggestion
      authority = Entities.create_authority(name: application_name, domain: application_base, config: metrics_config)
```

Reply:
@viv-4 should we revert this back to application_name now that Entities will use application_base if name isn't present?

Reply:
Actually application_name is incorrect here, it is for applictions and defaults to `backoffice`
see: https://github.com/PlaceOS/init/issues/94